src/state/account/accountService.ts

```diff
@@ -50,7 +56,18 @@ export const getAuthURLs = () =>
     })
   );
 
-export const fetchProfile = (params, currentUser) => {
+type UserQuery = {
+  users: {
+    edges: {
+      node: UserFragment & UserPreferencesFragment;
```

Comment:
question: what does the `&` do here?

Reply:
i will explain more in the intro later, but it is an intersection type operator, so a type which has the fields of both its operands